reflex/vars.py

```diff
@@ -1958,6 +1963,43 @@ def _determine_var_type(self) -> Type:
         return Any
 
 
+def computed_var(
+    fget: Callable[[BaseState], Any] | None = None,
+    fset: Callable[[BaseState, Any], None] | None = None,
+    fdel: Callable[[BaseState], Any] | None = None,
+    doc: str | None = None,
```

Comment:
can we remove these from the signature, computed vars never have setters or deleters, they're just in the property signature... actually probably should remove them from `__init__` as well if we're cleaning this up